imapd.conf

```diff
@@ -123,6 +123,7 @@ tls_session_timeout: 1440
 ##JMAP
 conversations: 1
 conversations_db: twoskip
+jmapsubmission_deleteonsend: 0
```

Comment:
On James side, we don't plan to store email submission

Reply:
Not even in memory?

Reply:
No not even.

What for? Memory won't be fitting per request load balancing in a distributed environment.